exercises/concept/rpn-calculator-inspection/.docs/instructions.md

```diff
@@ -0,0 +1,85 @@
+# Instructions
+
+Your work at _Instruments of Texas_ on an experimental RPN calculator continues. Your team has built a few prototypes that need to undergo a thorough inspection, to choose the best one that can be mass-produced.
```

Comment:
I love this _Instruments of Texas_ hook 💙 😆

Reply:
The fact that it had to be an RPN calculator to continue the theme complicated the test suite a bit :D but I really wanted to continue the theme 😎